### PR TITLE
add proxy support

### PR DESCRIPTION
**Pull request by Clemv95**
Adding support to proxy

### CODE REVIEW

**Comment by tarraschk** [comment]
@fwininger faut-il demander des tests unitaires ?

File: cbw_api_toolbox/cbw_api.py [hunk 1]
```diff
@@ -80,13 +82,23 @@ def _request(self, verb, payloads, body_params=None, params=None):
...
 
         try:
-            return requests.request(
-                verb,
-                route,
-                data=body_params,
-                params=params,
-                auth=CBWAuth(self.api_key, self.secret_key),
-                verify=self.verify_ssl)
+            if self.proxies != None:
```
Comment (fwininger) [inline cbw_api_toolbox/cbw_api.py:85]: I thinks that request use kwargs, so the if / else a not mandatory.

Comment (Clemv95) [reply cbw_api_toolbox/cbw_api.py:85]: Yeah probably

Comment (hezanathos) [reply cbw_api_toolbox/cbw_api.py:85]: Thanks for the PR. Can you edit to remove the condition ? This line adds as well style issues : 
cbw_api_toolbox/cbw_api.py:85:12: R1705: Unnecessary "else" after "return", remove the "else" and de-indent the code inside it (no-else-return)
cbw_api_toolbox/cbw_api.py:85:15: C0121: Comparison 'self.proxies != None' should be 'self.proxies is not None' (singleton-comparison)

Comment (Clemv95) [reply cbw_api_toolbox/cbw_api.py]: I've done it, is it good ? Or should i create a new PR ?

**Review by fwininger — APPROVED** [review]
looks good to me

**Pull request merged**